DZ1_var11.py

```diff
@@ -0,0 +1,95 @@
+# Вариант 11
+# Коэффициенты системы линейных уравнений заданы в виде прямоугольной матрицы.
+# С помощью допустимых преобразований привести систему к треугольному виду.
+# Найти количество строк, среднее арифметическое элементов которых, меньше заданной величины.
+import random
+
+
+def get_rnd_matrix(min_val, max_val, row, col):
+    return [[random.randint(min_val, max_val) for _ in range(col)] for _ in range(row)]
+
+
+def validate_matrix(matrix):
+    col_max = len(matrix[0])
+
+    # перевірка даних
+    if len(matrix) > len(matrix[0]):
+        return False
+
+    for row in matrix:
+        if col_max != len(row):
+            return False
+        for col in row:
+            if not isinstance(col, (int, float)):
+                return False
+    return True
+
+
+def print_matrix(matrix):
+    for row in matrix:
+        for col in row:
+            print(str(col).rjust(8), end='')
+        print('')
+
+
+def get_triangular_matrix(matrix):
+    row_max = len(matrix)
```

Comment:
the name **row_max** is not represent the content correctly. in the variable you have number of rows, so call the var in similar way